src/controllers/v1/chef_controller.ts

```diff
@@ -0,0 +1,128 @@
+import { Request, Response } from "express";
+import {
+  findAllChefs,
+  findChefById,
+  addChef,
+  updateChef,
+  removeChef,
+  findAllChefDishes,
+  findAllChefRestaurants,
+  
+} from "../../handlers/chef_handler";
+
+const getAllChefs = async (req: Request, res: Response) => {
+  try {
+    const chefs = await findAllChefs();
+    res.json(chefs);
+  } catch (error) {
+    res.status(500).json({ error: "Unable to fetch chefs." });
+  }
+};
+
+// const queryChef = async  (req: Request, res: Response) => {
+//   console.log("here");
+//   try {
+//     const value = req.query.value;
+//     console.log(value);
+//     const chefs = await findChefByQuery(value);
+//     res.json(chefs);
+//   } catch (error) {
+//     console.log(error);
+//     res.status(500).json({ error: "Unable to fetch chefs." });
+//   }
+// }
+const getChefByID = async (req: Request, res: Response) => {
+  const chefId = req.params.id;
+  try {
+    const chef = await findChefById(chefId);
```

Comment:
and if chefId could be undefined, what will happen?

Reply:
it will throw an error, I this that what we should expect